scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -2195,8 +2195,8 @@ function main {
                       ;;
                     1) getAnswerAnyCust relay_ip_enter "Enter relays's IPv4/v6 address"
                       if [[ -n "${relay_ip_enter}" ]]; then
-                        if ! isValidIPv4 "${relay_ip_enter}" && ! isValidIPv6 "${relay_ip_enter}"; then
-                          println ERROR "${FG_RED}ERROR${NC}: invalid IPv4/v6 address format!"
+                        if ! isValidIPv4 "${relay_ip_enter}" && ! isValidIPv6 "${relay_ip_enter}" && ! isValidHostnameOrDomain "${relay_ip_enter}"; then
```

Comment:
As ipv4 check is now split in actutal ipv4 and dns check, this needs to be reflected in the if statement a couple of lines below as well, else the relay is never added to the array.

Reply:
Based on this I believe the answer to 2b is:
* yes, option `[i]` should continue to accept hostnames (or domainnames) as a valid IPv4 address instead of requiring an actual IPv4 address.
Would the answer to 2a then be:
* No, do not validate hostname of domainname for option `[d]`, continue to accept any

Reply:
Though we do want some sanity checks, the main purpose here isn't to prevent people from being stupied and input incorrect data, but to know if we should use `--pool-relay-ipv4` or `--pool-relay-ipv6` directive. We could either add a new question for ipv4 vs ipv6 and IP vs DNS input, to know exact purpose of input and validate it accordingly. Or just be lazy like now and check if it has the format of an ipv6 address, else just use ipv4. 

Not sure whats preferred. Feel free to chime in what you think is the proper way to do it. Adding too many questions might be annoying as well.

Reply:
Agreed, too many questions may be annoying. 

I'm good with keeping the same behavior. I only wanted to clarify the existing behavior in case there was a preference to change it.